src/droplet.F

```diff
@@ -356,7 +359,7 @@ subroutine droplet_driver(dt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,
 #else
     !JMD WARNING: Loop does not yet match CPU version.
     !$acc parallel default(present)
-    !$acc loop gang vector reduction(+:num_fallout) &
+    !$acc loop gang vector reduction(+:num_fallout) reduction(max:errcode) &
```

Comment:
Do we need `errcode` besides debugging purpose?

Reply:
@sjsprecious, I think we do need to keep errcode variable.  Currently, if a bug occurs in the section, it is nearly impossible to locate the source of the bug due to the high volume of output generated from each CUDA thread.

Reply:
Gotcha! Thanks for the clarification.